pkg/api/validation/validation.go

```diff
@@ -612,9 +612,10 @@ func ValidateMinion(minion *api.Node) errs.ValidationErrorList {
 func ValidateMinionUpdate(oldMinion *api.Node, minion *api.Node) errs.ValidationErrorList {
 	allErrs := errs.ValidationErrorList{}
 
-	if !api.Semantic.DeepEqual(minion.Status, api.NodeStatus{}) {
-		allErrs = append(allErrs, errs.NewFieldInvalid("status", minion.Status, "status must be empty"))
-	}
+	// TODO: enable the code once we have better api object.status update model
```

Comment:
This makes it possible for anyone to write the node status?  Please add a comment to that effect, if true.